dashboards/magma-nodebucket.json

```diff
@@ -651,111 +535,772 @@
       "targets": [
         {
           "editorMode": "code",
-          "expr": "rate(kv_ep_magma_compactions{bucket=\"$bucket\"}[20s])",
-          "hide": false,
-          "legendFormat": "compactions",
+          "exemplar": true,
+          "expr": "rate(kv_ep_magma_write_bytes_bytes{bucket=\"$bucket\"}[2m])",
+          "interval": "",
+          "legendFormat": "total",
           "range": true,
           "refId": "A"
         },
         {
           "editorMode": "code",
-          "expr": "rate(kv_ep_magma_keyindex_compactions{bucket=\"$bucket\"}[20s])",
+          "expr": "rate(kv_ep_magma_write_bytes_compact_bytes{bucket=\"$bucket\"}[2m])",
           "hide": false,
-          "legendFormat": "key",
+          "legendFormat": "compactions",
           "range": true,
           "refId": "B"
         },
         {
           "editorMode": "code",
-          "expr": "rate(kv_ep_magma_seqindex_compactions{bucket=\"$bucket\"}[20s])",
-          "hide": false,
-          "legendFormat": "seq",
+          "exemplar": true,
+          "expr": "rate(kv_magma_write_bytes_{for=\"keyindex\", bucket=\"$bucket\"}[2m])",
```

Comment:
I don't see this metric. Is it not `kv_ep_magma_write_bytes_bytes`. This seems to be the case with many of the metrics referenced in this change. 

Also: I don't see the label "for" but maybe I'm looking at the wrong metric.

Can you review all the metrics you reference in this change?

Reply:
Thanks for the review!
I caught two metrics that weren't correctly named: kv_ep_magma_keyindex_compactions, kv_ep_magma_seqindex_compactions. These are now part of the Prometheus stat family "kv_magma_compactions" and are tagged with the label for=keyindex/seqindex.

Rest of them are correct and show up in the dashboard. These labels and new stats were recently added in kv-engine as part of [this](https://review.couchbase.org/c/kv_engine/+/193734) commit. So you'll have to be on latest master.